src/VM/Handler/Opcode8138Handler.cpp

```diff
@@ -40,9 +42,40 @@ namespace Falltergeist
 
             void Opcode8138::_run()
             {
+                //TODO: search ContainerItemObject and CritterObject inventories recursively if it has container itemobjects in it
+                //for now recursive inventories are not yet implemented since ItemObject has no inventory vector
+                
                 Logger::debug("SCRIPT") << "[8138] [=] int item_caps_total(void* obj)" << std::endl;
-                _script->dataStack()->popObject();
-                _script->dataStack()->push(0);
+                auto object = _script->dataStack()->popObject();
+                if (auto container = dynamic_cast<Game::ContainerItemObject*>(object))
+                {
+                    for(auto item: *container->inventory())
+                    {   
+                        if(item->PID() == 519)
```

Comment:
the only other ways i can think of doing it is an enum full of all the item pid's and reference it from this enum, but bottlecaps pid would still be hardcoded, just in an enum

The other way would be searching through the inventory vector of items by name, which would probably be pretty slow if it needs to do that often and the item name would be hardcoded.

How would i check how the original engine did it?